src/components/TracePage/TraceTimelineViewer/index.js

```diff
@@ -198,10 +198,8 @@ TimelineRow.Left.propTypes = {
 TimelineRow.Right = props => {
   const { children, ...rest } = props;
   return (
-    <div className="col-xs-9" {...rest}>
-      <div className="relative" style={{ height: '100%' }}>
```

Comment:
On the 

```
<div className="relative" style={{ height: '100%' }}>
```

It was necessary (due to the `relative`). But, it is not needed on the `<div className="col-xs-9 relative" {...rest}>`